packages/ui/src/components/Card/ProjectCard.tsx

```diff
@@ -94,3 +136,14 @@ const Row = styled.div`
     margin-top: 0;
   }
 `;
+
+const Button = styled.button`
+  width: 103px;
+  height: 44px;
+  border-radius: 8px;
+  // background-color: ${theme.palette.White}
+  // color: ${theme.palette.Navy}
+  background-color: #ffffff;
+  color: #31314e;
```

Comment:
요기서 궁금한게 있는데요...! 왜 위에걸로 하면 스타일이 안 먹을까용...

<img width="398" alt="image" src="https://github.com/CEOS-Developers/CEOS-FE/assets/65931227/e61ba8fe-a966-4dbc-8971-47850aa84eaa">

주석처리된 스타일로 먹이면 이렇게 되네요 ㅎ
다른 태그들은 스타일이 잘 먹던데.. button 태그라서 그런걸까요

Reply:
음 방금 브랜치 따서 확인해보니까 저는 잘 먹히는데 혹시 `;`를 속성 뒤에 추가안해줘서 그런거 아닌가요?

```suggestion
  background-color: ${theme.palette.White};
  color: ${theme.palette.Navy};
```

Reply:
ㅋㅋㅋㅋㅋㅋ 저는 바봅니다.. 바로 적용했어요 감사합니다 ㅎㅎ